build_projects/dotnet-cli-build/RemoveAssetFromDepsPackages.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Runtime.InteropServices;
+using System.Net.Http;
+using Microsoft.Build.Utilities;
+using Microsoft.DotNet.Cli.Build.Framework;
+using Microsoft.DotNet.InternalAbstractions;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+using static Microsoft.DotNet.Cli.Build.Framework.BuildHelpers;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class RemoveAssetFromDepsPackages : Task
+    {
+        public string DepsFile { get; set; }
+
+        public string SectionName { get; set; }
+
+        public string AssetPath { get; set; }
+
+        public override bool Execute()
+        {
+            DoRemoveAssetFromDepsPackages(DepsFile, SectionName, AssetPath);
+
+            return true;
+        }
+
+        private static void DoRemoveAssetFromDepsPackages(string depsFile, string sectionName, string assetPath)
```

Comment:
This code was copied from other code in the repo.  Can you delete the old code and have it call into this new method?  That way it will be less code that we have to clean up in the future.